app/helpers/layout_helper.rb

```diff
@@ -353,15 +353,16 @@ def base_errors_for(obj)
   end
 
   def popover(title, msg, options = {})
-    options[:icon] ||= 'info-sign'
-    content_tag(:a, icon_text(options[:icon], title), { :rel => "popover",
-                                                        :data => { :content => msg,
-                                                                   :"original-title" => title,
-                                                                   :trigger => "focus",
-                                                                   :container => 'body',
-                                                                   :html => true },
-                                                        :role => 'button',
-                                                        :tabindex => '-1' }.deep_merge(options))
+    options[:icon] ||= 'info-circle'
+    options[:kind] ||= 'fa'
```

Comment:
Default to kind `pficon`, icon `info` instead, that way it looks more homogeneous (see the status icons for instance)